src/sidebar/components/test/login-prompt-panel-test.js

```diff
@@ -0,0 +1,64 @@
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+
+import LoginPromptPanel from '../login-prompt-panel';
+import { $imports } from '../login-prompt-panel';
+
+import { checkAccessibility } from '../../../test-util/accessibility';
+import mockImportedComponents from '../../../test-util/mock-imported-components';
+
+describe('LoginPromptPanel', function() {
+  let fakeOnLogin;
+  let fakeOnSignUp;
+
+  let fakeStore;
+
+  function createComponent(props) {
+    return mount(
+      <LoginPromptPanel
+        onLogin={fakeOnLogin}
+        onSignUp={fakeOnSignUp}
+        {...props}
+      />
+    );
+  }
+
+  beforeEach(() => {
+    fakeStore = {
+      isLoggedIn: sinon.stub().returns(false),
+    };
+
+    fakeOnLogin = sinon.stub();
+    fakeOnSignUp = sinon.stub();
+
+    $imports.$mock(mockImportedComponents());
+    $imports.$mock({
+      '../store/use-store': callback => callback(fakeStore),
+    });
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  it('should render if user not logged in', () => {
```

Comment:
These tests are really lightweight, but I couldn't think of anything else material that needs testing.